03week/pigLatin.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+
+
+function pigLatin() {
+  
+    let word = document.getElementById("myString").value;
+  
+    //converts input word to lowercase
+    word = word.toLowerCase();
+  
+    //array of vowels
+    const vowelArr = ["a", "e", "i", "o", "u", "y"];
+  
+    //variable to hold letters of word until vowel
+    let vowel = "";
+  
+  
+    //get word count of input
+    //creates array of words splitting at any spaces
+    let words = word.split(' ');
+    //removes empty strings from 'words' array
+    words = words.filter(Boolean);
+    //counts words in 'words' array
+    let wordCount = words.length;
+  
+    //define empty string for translated sentence
+    let translated = "";
+    
+  
```

Comment:
Again, I would say that using too much empty lines between everything actually makes it harder to read.
Its somewhat personal preference, but I would recommend trying to find a balance.
Also try to make sure your code blocks are properly indented otherwise it gets hard to tell where your curly braces match up